docs/docs-source/docs/modules/get-started/pages/how-to-install-and-use-strimzi.adoc

```diff
@@ -0,0 +1,93 @@
+= How to install Strimzi and create a Kafka cluster
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+include::ROOT:partial$include.adoc[]
+
+This document shows how to install Strimzi and create a Kafka cluster suitable for use with Cloudflow.
+
+== Prerequisites
+
+=== CLI Tools
+
+Make sure you have the following prerequisites installed before continuing:
+
+- Helm, version 3 or later
+- Kubectl 
+
+== Installing Strimzi
+
+Strimzi can be installed using Helm. 
+
+
+== Creating a Kafka cluster using Strimzi
+
+To create a Kafka cluster using Strimzi we have to create a `CustomResource` of the kind `Kafka` in the namespace `cloudflow`. Creating the resource in the right namespace is important since we told Strimzi to only watch the `cloudflow` namespace when installing.
```

Comment:
Just a note: I don't think this is needed when we have finished the  Kafka AdminClient work, and not use the entity operator anymore.

Reply:
Do you mean the part of the namespace? Yea, I'm working on testing without the topic operator now, will change when tested.